### PR TITLE
Rename arguments in `stdlib/internals.ncl`

### PR DESCRIPTION
**Pull request by vkleen**
Move away from single letter or abbreviated argument names in `stdlib/internals.ncl` for better readability.

### CODE REVIEW

File: stdlib/internals.ncl [hunk 1]
```diff
@@ -3,50 +3,50 @@
...
+          (fun x => %assume% codomain (%go_codom% label) (value (%assume% domain (%chng_pol% (%go_dom% label)) x)))
       else
-          %blame% l,
+          %blame% label,
 
-  "$forall_var" = fun sy pol l t =>
-      let lPol = %polarity% l in
-      if pol == lPol then
-          %unseal% sy t (%blame% l)
+  "$forall_var" = fun sealing-key polarity label value =>
```
Comment (ebresafegaga) [inline stdlib/internals.ncl:28]: I didn't know Nickel variable names could have a `-` in them 😅

**Review by ebresafegaga — APPROVED** [review]
LGTM

File: stdlib/internals.ncl (hunk 1 above)
Comment (vkleen) [reply stdlib/internals.ncl:28]: Not sure if that fits the coding style of stdlib right now, but I felt like kebab 😛

Comment (yannham) [reply stdlib/internals.ncl:28]: @ebresafegaga Yup :slightly_smiling_face: it was made to accommodate for potential target configurations that use  options with this style. So it's typically rather for exported record fields (whose name is not chosen by you), but all identifiers have the same syntax, so you can indeed use a dash inside a function variable or a let-binding.

**Review by yannham — APPROVED** [review]

**Pull request merged**